### PR TITLE
Skip cgroups tests under WSL

### PR DESCRIPTION
**Pull request by hglkrijger**
- adds a skip predicate for WSL

### CODE REVIEW

File: tests/common/test_cgroups.py [hunk 1]
```diff
@@ -62,7 +63,10 @@ def path_maker(hierarchy, _):
 def i_am_root():
     return os.geteuid() == 0
 
+def is_wsl():
+    return '-Microsoft-' in platform.platform()
 
+@skip_if_predicate_true(is_wsl, "CGroups not supported under WSL")
```
Comment (jasonzio) [inline tests/common/test_cgroups.py]: All this does is suppress the test; it doesn't keep the agent code itself from trying to do the CGroup thing. The better fix would be to arrange for CGroups.enabled to always be False if it's running under WSL.

Is there an osutil subclass for WSL? If we created one, that would be the place to insert the necessary code to cause CGroups functionality to be completely disabled (including the tests).

Comment (jasonzio) [reply tests/common/test_cgroups.py]: In osutil/ubuntu.py:
```
    def is_cgroups_supported(self):
        if ('TRAVIS' in os.environ and os.environ['TRAVIS'] == 'true') \
                or ('-Microsoft-' in platform.platform()):
            return False
        return True
```

Comment (hglkrijger) [reply tests/common/test_cgroups.py]: Ok, updated

File: azurelinuxagent/common/osutil/ubuntu.py [hunk 2]
```diff
@@ -56,9 +57,9 @@ def get_dhcp_lease_endpoint(self):
         return self.get_endpoint_from_leases_path('/var/lib/dhcp/dhclient.*.leases')
 
     def is_cgroups_supported(self):
-        if 'TRAVIS' in os.environ and os.environ['TRAVIS'] == 'true':
-            return False
-        return True
+        is_wsl = '-Microsoft-' in platform.platform()
```
Comment (boumenot) [inline azurelinuxagent/common/osutil/ubuntu.py]: Nice.

**Pull request merged**